pkgs/development/python-modules/ciscoconfparse/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchFromGitHub, passlib, dnspython, loguru, toml
+, ipaddr, poetry, poetry-core, black, Fabric, pytest, sphinx }:
+
+buildPythonPackage rec {
+  pname = "ciscoconfparse";
+  version = "1.6.21";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "mpenning";
+    repo = pname;
+    rev = version;
+    sha256 = "1d6nzhmdg7zlg1h3lm4v7j4hsb2aqd475r5q5vcqxfdxszc92w21";
+  };
+
+  patchPhase = ''
+    patchShebangs tests
+  '';
+
+  propagatedBuildInputs =
+    [ passlib dnspython loguru toml ipaddr poetry black Fabric sphinx ];
```

For consistency as well:
```suggestion
  propagatedBuildInputs = [ passlib dnspython loguru toml ipaddr poetry black Fabric sphinx ];
```
Or
```suggestion
  propagatedBuildInputs = [
    passlib
    dnspython
    ...
  ];
```